### PR TITLE
Instant cameras can now be used as circuit shells.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
A circuit and shell components have been given to instant cameras around the station, so they can now be used for wiremod. There's no new wiremod design here since they can already be printed at any autolathe.
@Watermelon914 also told me a while ago that items should be given the shell component and not the usb port one. 

## Why It's Good For The Game
Outside of the sci department and more specifically the protolathe and component printer, there's currently a lack of objects that can be used as circuit shells, which means wiremod is more or less limited to its own printables and the less versatile, higher end usb ports around the station. This is a small step toward a more inclusive wiremod that can use station devices for circuits building. 

## Changelog

:cl:
expansion: Handheld cameras (not the wallmounted security ones) can now be used as circuit shells.
/:cl:

### CODE REVIEW

**Comment by Mickyan** [comment]
Can you justify the addition of PICTURE_SIZE_CIRCUIT_BONUS? This seems needlessly artificial when we could have better brands of cameras or just buff their range instead of circuits magically making them better.

**Comment by Ghommie** [comment]
Golly, you're probably right. That's more of a supposed hardware, not software, limitation, so it doesn't make much sense for it to be circumverted through this sort of customization. I'll remove that though I don't have a plan to add better cameras right now

**Comment by datorangebottle** [comment]
any chance we'll be able to use "people contained in the picture" as a value, or send the picture to somewhere else using an ntnet transmitter? Just a couple examples of cool functionality that might be possible.

**Comment by Ghommie** [comment]
> 
> 
> any chance we'll be able to use "people contained in the picture" as a value, or send the picture to somewhere else using an ntnet transmitter? Just a couple examples of cool functionality that might be possible.

So far, the only atom output is the photographed entity, through either manual usage or the target input, or alternatively a turf with coords. I'm not sure about making it a list datatype since pictures may catch a lot of other movables which normally should be ignored, although this doesn't mean I'm against it.
Regarding a picture output, NTnet is tad barebones and I don't know of any device that can construe and utilize picture datums through it or circuits. I have some ideas about NTnet+circuits but none is concrete. Perhaps in a near future.

File: code/modules/photography/camera/camera.dm [hunk 1]
```diff
@@ -243,6 +247,70 @@
...
-		p.set_picture(picture, TRUE, TRUE)
-		if(CONFIG_GET(flag/picture_logging_camera))
-			picture.log_to_file()
+	p.set_picture(picture, TRUE, TRUE)
+	if(CONFIG_GET(flag/picture_logging_camera))
+		picture.log_to_file()
+
+/obj/item/circuit_component/camera
+	display_name = "Camera"
+	desc = "A polaroid camera that takes pictures when triggered."
```
Comment (Watermelon914) [inline code/modules/photography/camera/camera.dm:256]: make sure to clarify the coordinate system is relative and not global

**Pull request merged**